src/index.html

```diff
@@ -13,7 +13,26 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <div class="display-grid">
```

Comment:
don't use styles as class name, give the name like 'container' or 'box-container' or something like this